src/css/style.scss

```diff
@@ -204,180 +159,342 @@ button {
   justify-content: center;
   align-items: center;
 }
-.form-btn__submit {
+.form__submit {
   text-transform: uppercase;
   font-weight: 500;
-  padding: 1rem 2.5rem;
+  padding: 1rem 3.5rem;
   margin: 1.5rem;
   font-size: 1.3rem;
-  background: darken($color: $grey, $amount: 10%);
+  background: lighten($color: $grey, $amount: 2%);
 }
-#close {
-  background: darken($color: $grey, $amount: 5%);
+.form__close {
+  background: lighten($color: $grey, $amount: 15%);
 }
-.form-btn__close::before {
+.form__close::before {
   content: "\f100";
   display: block;
   position: relative;
   left: 20rem;
   bottom: 38rem;
 }
 
-//Gallery
+// Gallery
 .gallery {
   display: flex;
   flex-wrap: wrap;
   justify-content: space-between;
   //fixes the last row formating
   &::after {
     content: "";
-    width: 50rem;
+    width: 30rem;
+  }
+  .gallery-img__container {
+    width: 30rem;
+    height: 100%;
+    position: relative;
+  }
+  .gallery-img {
+    width: 100%;
+    display: block;
+    opacity: 1;
+    object-fit: cover;
   }
 }
 
-.gallery-img--container {
-  width: 50rem;
-}
-.gallery-img {
-  width: 100%;
-  height: 30rem;
-  display: block;
-  opacity: 1;
-  object-fit: cover;
-  transition: opacity 0.3s ease;
-  &:hover {
-    opacity: 0.8;
+.gallery__edit-btn {
+  display: none;
+  // display: block;
```

Comment:
@maya I'm not understanding what you are trying to archive with this animation.

This MR is focusing on too many things at once. DB refactor, new slideshow, Mansory, edit btn, ... let's go a step back, shall we? It gets confusing.

My recommendation: Finish the DB refactor and the slideshow, and merge it, without mansory stuff, just a simple layout. The JS is getting complex because of the extra DOM you are creating. I recommend doing the handlerbarjs first. it will simplify the JS and HTML part.

Reply:
👋 don't think you meant me